<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>545</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="input-keyboard">
    <normaloff>C:/Users/lenovo/.designer/backup</normaloff>C:/Users/lenovo/.designer/backup</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 500 10pt &quot;霞鹜文楷 中粗&quot;;</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="Browser1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>开始聊天吧~</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Edit1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>711</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>347</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
